bangazonapi/views/profile.py

```diff
@@ -355,6 +355,15 @@ class Meta:
 
 
 class RecommenderSerializer(serializers.ModelSerializer):
+    """JSON serializer for recommendations"""
+    customer = CustomerSerializer()
+    product = ProfileProductSerializer()
+
+    class Meta:
+        model = Recommendation
+        fields = ('product', 'customer',)
```

Comment:
Great work using these serializers to show one relationship model from two different perspectives.